tests/macsec/test_lacp_transparency.py

```diff
@@ -0,0 +1,208 @@
+'''
+
+The test case will verify that the applied Macsec sessions are secured
+even if LACP isn't up.
+
+1: Configure 2 portchannels on DUT with two members each.
+2: First ensure portchannels are up and macsec sessions are secured.
+3: Mismatch interfaces on both portchannels. LACP should go down while macsec
+sessions should stay secured.
+
+'''
+import pytest
+import logging
+import sys
+import time
```

Comment:
Thanks ! we can make this test simpler

1. Please add as a new test under test_interop_protocol.py
2. So when the test starts up the macsec sessions are already present 
3. Find one which is over portchannel.
4. Do the tests you would like to verify 
5. restore it back 

We need not do config reload correct ?

Also the please add this test in https://github.com/sonic-net/sonic-mgmt/blob/master/docs/testplan/MACsec-test-plan.md under the interop cases. No need to have a new .md file